src/ui/src/partials/request_view.html

```diff
@@ -94,7 +94,11 @@ <h1 class="page-header">
     <div class="row">
       <div class="col-md-6">
         <div style="margin-bottom: 10px;">
-          <span class="h3">Output</span>
+          <span class="h3">Output
+            <button type="button" class="btn pull-right" ng-click="download();">
```

Comment:
The semicolon after `download()` is not needed. We use that when calling straight JS. When referencing functions in the HTML tags, we have not been using it. Both ways still work.